Formula/wtfutil.rb

```diff
@@ -0,0 +1,72 @@
+class Wtfutil < Formula
+  desc "The personal information dashboard for your terminal"
+  homepage "https://wtfutil.com"
+  url "https://github.com/wtfutil/wtf/archive/v0.10.3.tar.gz"
+  sha256 "8afa15a5729bfa5a8d009e3d03423cf7472e797740b92e6f3e66ca8fd05c8454"
+
+  depends_on "go" => :build
+
+  resource "config.yml" do
+    url "https://gist.githubusercontent.com/chenrui333/746c939f4a30324f1bbc36c4be38c907/raw/8b5b192c3d50845457b7778e0000ffca5e87212b/config.yml"
+    sha256 "edfe1ee724a9625f0fb90f421faf0f0427bcd69183ddda04d24b064f9973951a"
+  end
+
+  def install
+    # ENV["GOPATH"] = buildpath
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
+
+    dir = buildpath/"src/github.com/wtfutil/wtf"
+    dir.install buildpath.children
+
+    cd dir do
+      system "go", "build", "-o", bin/"wtf"
+      prefix.install_metafiles
+    end
+  end
+
+  def caveats; <<~EOS
+    Please add a config.yml file to your ~/.config/wtf directory.
+    See https://github.com/wtfutil/wtf for details.
+    open /Users/rchen/.config/wtf/config.yml: no such file or directory
+  EOS
```

Comment:
This entire caveats block needs to be removed.

Reply:
👍